packages/expo-ui/components/Button/types.ts

```diff
@@ -0,0 +1,248 @@
+export type MaterialIcon =
```

Comment:
Follows format in https://github.com/expo/expo/pull/28845

Reply:
@alanjhughes We should consider how expo-ui relates to expo-symbols.

We could either move this logic to expo-symbols as in your draft and use a native component from expo-symbols under the hood, accept a <SymbolView> as a prop (confusing imo), move expo-symbols to expo-ui as an Icon component or just keep these separate.

WDYT?